index.html

```diff
@@ -7,14 +7,150 @@
     <!-- Here we import the Boostrap CSS stylesheet -->
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
     <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" href="./css/main.css">
+    <!-- Add icon library -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
     <title>Responsive grid project</title>
 </head>
 
+<!-- Header / Navigation Bar-->
 <body>
+    <header>
 
-    <!-- Add your markup here -->
+        <!-- Mobile Nav Bar-->
 
+        <nav class="mobile-navigation">
+            <div class="navbar-tops">
+                <button class="appointment-button"> Book an appointment</button>
+                <button class="menu-button" onclick="openMenu()">
+                    &#9776;
+                  </button>
+            </div>
+
+            <h1 class="phd-text text-center col-12 col-md-6">Maria Hope, Ph.D. - Psychologist</h1>
+            <div class="navbar-nav mobile-menu mobile-menu-hidden" id="mobile-menu">
+                <a href="#">Home</a>
+                <a href="#">About</a>
+                <a href="#">Services</a>
+                <a href="#">Blog</a>
+                <a href="#">Contact</a>
+              </div>
+        </nav>
+
+        <!-- Desktop Nav Bar-->
+
+        <nav class="desktop-navigation">
+            <div class="maria-phd">
+                 <h1 class="phd-text ">Maria Hope, Ph.D. - Psychologist</h1>
+            </div>
+
+            <div class="nav-bar-right">
+                <button class="appointment-button"> Book an appointment</button>
+                
+                <div class="bar-extended">
+                    <a href="#">Home</a>
+                    <a href="#">About</a>
+                    <a href="#">Services</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Contact</a>
+                  </div>
+            </div>
+        </nav>
+    </header>
+
+<!-- Main content -->
+
+    <main class="primary-content">
+        <section class="hero-main">
+            <img class="col-12" src="./img/suculent.webp" alt="Succulent plant">
+
+            <section class="mint-container col-10 offset-1 col-md-8 offset-md-2">
```

Comment:
Follow the Bootstrap rules: you're using `col-...` classes here without a `container` or a `row`.

Try applying `container-fluid` to the top level (either once on the `main`, or on each section individually). 
Then, always use `row` before you use `col-...`